### PR TITLE
WIP: Update variable usage

### LINKED ISSUE #15: Update font weight
The button has a weird font weight atm.

//cc @JPorry

### PR DESCRIPTION
**Pull request by jamesplease**
As part of #8 

Also resolves #15 

---

This begins the process of normalizing our variable usage. **It's not complete**, and is instead meant to serve as a reference point.

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,400,700,900">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700">
```
Comment (jamesplease) [inline .storybook/preview-head.html]: @JPorry in #15 , you mentioned that we use 300, 500, and 700. I think that 500 refers to the semibold weight, whereas 400 is the "normal" weight.

Typically, in my apps I use normal (400) most of the time, and then 500 for "bold" things. I think it'd be cool to include both 400 and 500 at least. I can't think of instances where I use 700, but I left it in here.

What do you think?

File: components/button/button.css [hunk 2]
```diff
@@ -61,7 +70,8 @@
 }
 
 .mt-button.mt-button-secondary {
```
Comment (jamesplease) [inline components/button/button.css]: I think we can get away with not increasing the specificity for these other selectors. i.e.; `.mt-button-secondary` by itself should be enough. What do you think @JPorry ?

File: .storybook/preview-head.html (hunk 1 above)
Comment (JPorry) [reply .storybook/preview-head.html]: yeah, makes sense. 300,400,500 sounds like enough.

File: components/button/button.css (hunk 2 above)
Comment (JPorry) [reply components/button/button.css]: Totally! I usually leave things with a specificity of one class, but there are some mistakes here and there that we need to detect and fix!
Thanks man!

File: components/checkbox/checkbox.css [hunk 3]
```diff
@@ -1,4 +1,16 @@
...
+    This (ugly) set of variables defines the variable fallback implementation.
+
+    1. If `mt-checkbox-backgroundColor` is set, then that will be applied
+    2. Otherwise, `mt-mainColor` will be applied
+    3. Otherwise, `#2196f3` will be applied
+  */
+  --_mt-checkbox-backgroundColor: var(--mt-checkbox-backgroundColor, var(--mt-mainColor, #2196f3));
+  --_mt-checkbox-checkColor: var(--mt-checkbox-checkColor, #fff);
+  --_mt-checkbox-borderRadius: var(--mt-checkbox-borderRadius, 2px);
+  --_mt-checkbox-borderWidth: var(--mt-checkbox-borderWidth, 2px);
```
Comment (JPorry) [inline components/checkbox/checkbox.css]: I am not sure about the borderRadius and the borderWidth being customizable. I think we should just allow to change colors for now. What do you think?

File: stories/chip.stories.js [hunk 4]
```diff
@@ -19,6 +19,6 @@ storiesOf('Chip', module)
       closeable={boolean('Closeable', false)}
       onCloseClick={action('CloseClicked')}
       onClick={action('Click')}>
-      {text('Text', 'Hello Button')}
+      {text('Text', 'Hello Chip')}
```
Comment (JPorry) [inline stories/chip.stories.js]: Copy/Paste FTW!

File: components/checkbox/checkbox.css (hunk 3 above)
Comment (jamesplease) [reply components/checkbox/checkbox.css]: Agreed. I'll remove them.

File: stories/chip.stories.js (hunk 4 above)
Comment (jamesplease) [reply stories/chip.stories.js]: 😂

**Pull request merged**